lua/nordic/tests/options.lua

```diff
@@ -1,55 +1,168 @@
--- First test the default config, and then test different variations of the config.
-
-local config = require('nordic.config').options
+local assert_eq = require('nordic.tests').assert_eq
 local load = require('nordic').load
+local U = require('nordic.utils')
+
+local function highlight(group)
```

Comment:
Nit: Maybe name this `get_highlight`.

It is a nice function, maybe move it to utils, even if it is only used here for now.

Reply:
[refactor(utils): extract highlight to utils.get_highlight;](https://github.com/AlexvZyl/nordic.nvim/pull/150/commits/d6b327848db302c5bf7991b6082fe690e5b2943e)
[fix(utils): rename highlight to apply_highlights;](https://github.com/AlexvZyl/nordic.nvim/pull/150/commits/967d745d2107d1ffcf8f2cfe8d998215a8d0f464)